File: Server/REST SERVER/src/main/java/com/entity/DishesAmount.java
package com.entity;

/**
 * Created by gusvl on 11.05.2017.
 */
public class DishesAmount {
    private int dishId;
    private int amount;

    public DishesAmount() {
    }

    public DishesAmount(int dishId, int amount) {
        this.dishId = dishId;
        this.amount = amount;
    }

    public int getDishId() {
        return dishId;
    }

    public void setDishId(int dishId) {
        this.dishId = dishId;
    }

    public int getAmount() {
        return amount;
    }

    public void setAmount(int amount) {
        this.amount = amount;
    }
}

File: iOS/Food Delivery/Food Delivery/Model/DishesAmount.m
//
//  DishesAmount.m
//  Food Delivery
//
//  Created by Kirill Budevich on 12.05.17.
//  Copyright © 2017 Kirill Budevich. All rights reserved.
//

#import "DishesAmount.h"

@implementation DishesAmount

+ (NSDictionary *)JSONKeyPathsByPropertyKey {
    return @{
             @"dishId": @"dishId",
             @"amount": @"amount"
             };
}

- (instancetype)initWithDishId:(NSNumber *)dishId amount:(NSNumber *)amount {
    
    if (self = [super init]) {
        self.dishId = dishId;
        self.amount = amount;
    }
    return self;
}

@end
